lib/tasks/import_prebakedstats.rake

```diff
@@ -0,0 +1,160 @@
+      :total_value_2010_baseline,
+      :total_value_2015,
+      :total_value_2016
+    ].freeze
+
+    iso3, name = nil
+    protected_value, protected_percentage = 0
+
+    parse_change_stat_hash.keys.each do |key|  
+      if key == :iso3
+        iso3 = csv_row[strip_key(parse_change_stat_hash[key])]&.strip
+      elsif key == :name
+        name = csv_row[strip_key(parse_change_stat_hash[key])]&.strip
+      elsif total_value_array.include? key
+        (total_value_years[key] ||= csv_row[strip_key(parse_change_stat_hash[key])]&.strip)
+      elsif key == :protected_value
+        protected_value = csv_row[strip_key(parse_change_stat_hash[key])]&.strip
+      elsif key == :protected_percentage
+        protected_percentage = csv_row[strip_key(parse_change_stat_hash[key])]&.strip
+      end
+    end
+
+    geo_entity = fetch_geo_entity(name, iso3)
+    insert_change_stat(habitat, geo_entity, total_value_years, protected_value, protected_percentage)
+  end
+
+  def parse_geo_entity_stat csv_headers, csv_row, geo_entity_type, habitat
+    parse_geo_entity_stat_hash = {
+      iso3: csv_headers[0],
+      total_value: csv_headers[1],
+      total_value_protected: csv_headers[2],
+      protected_percentage: csv_headers[3].chomp
+    }.freeze
+
+    iso3, name = nil
+    total_value, total_value_protected, protected_percentage = 0
+
+    parse_geo_entity_stat_hash.keys.each do |key|
+      if key == :iso3
+        iso3 = csv_row[strip_key(parse_geo_entity_stat_hash[key])]&.strip
+      elsif key == :name
+        name = csv_row[strip_key(parse_geo_entity_stat_hash[key])]&.strip
+      elsif key == :total_value
+        total_value = csv_row[strip_key(parse_geo_entity_stat_hash[key])]&.strip
+      elsif key == :total_value_protected
+        total_value_protected = csv_row[strip_key(parse_geo_entity_stat_hash[key])]&.strip
+      elsif key == :protected_percentage
+        protected_percentage = csv_row[strip_key(parse_geo_entity_stat_hash[key])]&.strip
+      end
+    end
+    geo_entity = fetch_geo_entity(name, iso3)
+    insert_geo_entity_stat(habitat, geo_entity, total_value, total_value_protected, protected_percentage)
+  end
+
+  def insert_geo_entity_stat(habitat, geo_entity, total_value, total_value_protected, protected_percentage)
+    habitat = Habitat.find_by(name: habitat)
+
+    puts "GeoEntity is: #{geo_entity&.name}, habitat is: #{habitat.name}"
+    ges = GeoEntityStat.create(habitat: habitat, geo_entity: geo_entity, protected_value: total_value_protected || 0, 
+                        total_value: total_value || 0, protected_percentage: protected_percentage || 0)
+
+    puts "GeoEntityStat is: #{ges.inspect}"
+    
+  end
+
+  def insert_change_stat(habitat, geo_entity, total_value_years, protected_value, protected_percentage)
+    habitat = Habitat.find_by(name: habitat)
+
+    puts "GeoEntity is: #{geo_entity&.name}, habitat is: #{habitat.name}"
+    cs = ChangeStat.create(habitat: habitat, geo_entity: geo_entity, 
```

Comment:
Here as well, you could use the assumption that csv headers should already be formatted followin the column names definitions.